### PR TITLE
Fixed info message for multiple users with common name

### PR DESCRIPTION
**Pull request by c0dzilla**

### CODE REVIEW

**Comment by c0dzilla** [comment]
@arpit270896 please review it. It's a very small change.

File: scripts/sdsinfo.coffee [hunk 1]
```diff
@@ -17,16 +17,20 @@ module.exports = (robot) ->
       response = JSON.parse body 
       details = -1
       if response["version"]
-       reply = sendTheMsg(row) for row in response.feed.entry
-       if details != -1
-        msg.send details
-       else 
-        msg.send "User Not found"
+        reply = sendTheMsg(row) for row in response.feed.entry
```
Comment (csoni111) [inline scripts/sdsinfo.coffee]: What's the use of this `reply` variable?

Comment (c0dzilla) [inline scripts/sdsinfo.coffee]: Removed it.

**Comment by c0dzilla** [comment]
@csoni111 when do you plan to fix the attachment message? Maybe it can be added in a separate pr as it is an additional feature while this one can address the small info bug for multiple users?

**Comment by csoni111** [comment]
Yeah you are right @c0dzilla, sending attachment messages will require the hubot version to be updated, which requires a lot of changes. That can be taken up later in another pr. For now removing the commit from this pr and merging it.

**Pull request merged**